tests/test-run.sh

```diff
@@ -433,6 +433,11 @@ else
         df --output=size --block-size=1K "$(pwd -P)" > dir-size
     assert_file_has_content dir-size '^ *1024$'
     echo "ok - tmpfs has expected size"
+else
+    $RUN --size 1048576 --tmpfs "$(pwd -P)" true
+    $RUN --perms 01777 --size 1048576 --tmpfs "$(pwd -P)" true
```

Comment:
Can you still use `stat` here to ensure the permissions are correct?

Reply:
Potentially yes, but that'll make the diff larger and noisier, and I wasn't sure that was a good tradeoff for something we're only doing for the benefit of pre-2013 or non-GNU environments.

Reply:
Right, you have a point.